ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -387,152 +357,116 @@ def check_backend_asds(logger, disks, backend_name, config):
                             named_params={'host': ip_address, 'port': str(disk.get('port')),
                                           'long-id': disk_asd_id},
                             extra_params=[key])
             except ObjectNotFoundException:
-                defectivedisks.append(disk_asd_id)
-                logger.failure("ASD test with DISK_ID '{0}' failed on NODE '{1}'!"
-                               .format(disk_asd_id, ip_address),
-                               'alba_asd_{0}'.format(disk_asd_id))
+                broken_disks.append(disk_asd_id)
+                result_handler.failure('ASD test with DISK_ID {0} failed on NODE {1}!'.format(disk_asd_id, ip_address), test_name)
             except (AlbaException, DiskNotFoundException) as e:
-                defectivedisks.append(disk_asd_id)
-                logger.failure("ASD test with DISK_ID '{0}' failed because: {1}"
-                               .format(disk_asd_id, str(e)),
-                               'alba_asd_{0}'.format(disk_asd_id))
-        return workingdisks, defectivedisks
+                broken_disks.append(disk_asd_id)
+                result_handler.failure('ASD test with DISK_ID {0} failed because: {1}'.format(disk_asd_id, str(e)), test_name)
+        return result
 
+    # @todo: separate cluster-wide-check
     @staticmethod
-    @expose_to_cli('alba', 'backend-test')
-    def check_backends(logger):
+    @expose_to_cli(MODULE, 'backend-test')
+    def check_backends(result_handler):
         """
         Checks Alba as a whole
 
-        :param logger: logging object
-        :type logger: ovs.log.healthcheck_logHandler.HCLogHandler.HCLogHandler
+        :param result_handler: logging object
+        :type result_handler: ovs.extensions.healthcheck.results.HCResults
         """
-
-        logger.info("Checking available ALBA backends ...")
+        test_name = '{0}-backend-test'.format(AlbaHealthCheck.MODULE)
+        result_handler.info('Checking available ALBA backends.')
         try:
-            alba_backends = AlbaHealthCheck._fetch_available_backends(logger)
+            alba_backends = AlbaHealthCheck._fetch_available_backends(result_handler, test_name)
             if len(alba_backends) == 0:
-                return logger.skip("No backends found ...", 'alba_backends_found')
+                return result_handler.skip('No backends found.', test_name)
 
-            logger.success("We found {0} backend(s)!".format(len(alba_backends)),
-                           'alba_backends_found'.format(len(alba_backends)))
+            result_handler.success('We found {0} backend(s)!'.format(len(alba_backends)), test_name)
 
-            logger.info("Checking the ALBA ASDs ...")
+            result_handler.info('Checking the ALBA ASDs.')
             for backend in alba_backends:
+                backend_name = backend['name']
                 # check disks of backend, ignore global backends
                 if backend.get('type') != 'LOCAL':
```

Comment:
`type` will be in the dictionary, so no need for get
If you'd ever change the code to not include `type` you'd be seeing GLOBAL backends all over the place, which would be incorrect